dspace-api/src/main/java/org/dspace/core/Context.java

```diff
@@ -129,7 +129,6 @@
     }
 
     protected Context(EventService eventService, DBConnection dbConnection) {
-        this.mode = Mode.READ_WRITE;
```

Comment:
I removed the default value to avoid setting the mode again in case a new context is initialized